### PR TITLE
[SPARK-21880][web UI]In the SQL table page, modify jobs trace information

### PR DESCRIPTION
**Pull request by Geek-He**
## What changes were proposed in this pull request?
As shown below, for example, When the job 5 is running, It was a mistake to think that five jobs were running, So I think it would be more appropriate to change jobs to job id.
image

## How was this patch tested?
no need

### CODE REVIEW

File: sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala [hunk 1]
```diff
@@ -177,7 +177,7 @@ private[ui] class RunningExecutionTable(
     showFailedJobs = true) {
 
   override protected def header: Seq[String] =
-    baseHeader ++ Seq("Running Jobs", "Succeeded Jobs", "Failed Jobs")
+    baseHeader ++ Seq("Running Job Id", "Succeeded Job Id", "Failed Jobs Id")
```
Comment (srowen) [inline sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala]: OK, but these should be "Job IDs"

**Comment by srowen** [comment]
CC @zsxwing

File: sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala (hunk 1 above)
Comment (Geek-He) [reply sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala]: I've changed it this way. But I don't understand why is IDs, not ID?There are no multiple JOB common execution plans.

**Comment by ajbozarth** [comment]
I'm ok with this change

**Comment by zsxwing** [comment]
ok to test

**Comment by zsxwing** [comment]
LGTM pending tests

**Comment by zsxwing** [comment]
Thanks! Merging to master.

**Pull request closed without merging**